chrome/content/zotero/bindings/itembox.xml

```diff
@@ -2476,6 +2490,8 @@
 							return !hideTransforms;">
 						<menuitem label="&zotero.item.creatorTransform.nameSwap;"
 							oncommand="document.getBindingParent(this).swapNames();"/>
+						<menuitem label="&zotero.item.textTransform;"
+							oncommand="document.getBindingParent(this).textTransform(document.popupNode, 'title')"/>
```

Comment:
Well, we could actually have the "Transform Text" menu and "Title Case" within it, but if that's always going to be the only option maybe just "Convert to Title Case"? "Title Case" could technically be seen as an imperative verb, but it seems a little more awkward to have just that.